epictrack-web/src/components/project/ProjectForm.tsx

```diff
@@ -54,6 +54,18 @@ const schema = yup.object<Project>().shape({
     .required()
     .min(-180, "Longitude must be greater than or equal to -180")
     .max(180, "Longitude must be less than or equal to 180"),
+  capital_investment: yup
+    .number()
+    .typeError("Please provide a numerical value")
+    .nullable(),
+  fte_positions_construction: yup
```

Comment:
There is another simplest way by which you can enforce numerical value entry by setting the type="number" in the control. I have done it in number_of_days field in task in work plan